ui/app/styles/components/spinner.scss

```diff
@@ -0,0 +1,25 @@
+.spinner {
+  position: relative;
+
```

Comment:
nit: blank line

Reply:
Whoops, missed this before I merged. Sorry!